lib/improver/cli/temp_lapse_rate.py

```diff
@@ -82,40 +82,44 @@ def process(temperature: cli.inputcube,
         ValueError: If minimum lapse rate is greater than maximum.
         ValueError: If Maximum height difference is less than zero.
         ValueError: If neighbourhood radius is less than zero.
-
+        RuntimeError: If calculating the true lapse rate and orography or
+                      land mask arguments are not given.
     """
     import numpy as np
     from improver.lapse_rate import LapseRate
     from improver.metadata.amend import amend_attributes
 
-    attributes_dict = {"title": "remove", "source": "remove",
+    new_metadata = {"title": "remove", "source": "remove",
                        "history": "remove", "um_version": "remove"}
 
-    if return_dalr:
-        result = temperature.copy(
-            data=np.full_like(temperature.data, U_DALR.points[0]))
+    if dry_adiabatic:
+        result = temperature.copy(data=np.full_like(temperature.data, DALR))
         result.rename('air_temperature_lapse_rate')
-        result.units = U_DALR.units
-        amend_attributes(result, attributes_dict)
+        result.units = U_DALR
+        amend_attributes(result, new_metadata)
         return result
 
     if min_lapse_rate > max_lapse_rate:
         msg = 'Minimum lapse rate specified is greater than the maximum.'
         raise ValueError(msg)
 
-    if max_height_diff_metres < 0:
+    if max_height_diff < 0:
         msg = 'Maximum height difference specified is less than zero.'
         raise ValueError(msg)
 
     if nbhood_radius < 0:
         msg = 'Neighbourhood radius specified is less than zero.'
         raise ValueError(msg)
 
+    if orography is None or land_sea_mask is None:
+        msg = 'Missing orography and/or land mask arguments.'
+        raise RuntimeError(msg)
```

Comment:
Do you need to add a test for this. I think all the ValueErrors this CLI currently raises are tested.

Reply:
Added

Reply:
Thanks